src/lib.rs

```diff
@@ -1948,21 +1948,6 @@ impl Build {
                                 .into(),
                             );
                         }
-                    } else if target.contains("aarch64-apple-tvos") {
-                        if let Some(arch) =
-                            map_darwin_target_from_rust_to_compiler_architecture(target)
-                        {
-                            let sdk_details =
-                                apple_os_sdk_parts(AppleOs::TvOs, &AppleArchSpec::Device(""));
-                            let deployment_target = self.apple_deployment_version(
-                                AppleOs::TvOs,
-                                None,
-                                &sdk_details.sdk,
-                            );
-                            cmd.args.push(
-                                format!("--target={}-apple-tvos{}", arch, deployment_target).into(),
-                            );
```

Comment:
This change seems to misalign with XCode at least. When building Objective-C with `clang` via XCode it passes the `-target` flag to convey the deployment target: `-target arm64-apple-tvos16.0` instead of the `os-version-min` flag we pass (which doesn't seem documented for tvOS/watchOS? ignoring that though...)

Did you try building any libs with this just to make sure?

Reply:
That is a good point. I will add back this code as everything worked fine either way and that will align it with XCode's behavior.
As for libraries that I built this on there are two. [Blake3](https://github.com/BLAKE3-team/BLAKE3) and [Ring](https://github.com/briansmith/ring). Both are dependencies of the project I am working on [Ditto](https://github.com/getditto/DittoSwiftPackage) which is included in my iPad/tvOS application/s.